perfmetrics/scripts/continuous_test/ml_tests_tf/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Installing docker engine
+echo "Installing linux utility packages, like, lsb-release, curl..."
+sudo apt-get update
+sudo apt-get install -y \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+
+echo "Installing docker framework..."
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/"
+
+# Using the script from dlc_testing branch
+mkdir dlc_setup
+cd dlc_setup
+
+mkdir container_artifacts && mkdir container_artifacts/logs && mkdir container_artifacts/output
+
+# Building the dlc based dockerfile
+git clone https://github.com/GoogleCloudPlatform/gcsfuse.git
+cd gcsfuse
+git checkout dlc_testing
+sudo docker build . -f perfmetrics/dlc/Dockerfile -t dlc-gcsfuse --build-arg DLC_IMAGE_NAME=tf-gpu.2-10
```

Comment:
It is better to keep as much code as possible in master branch and only use dlc_testing branch till the log rotation feature is not added to master branch.
So, It would be better to organise this code similar to https://github.com/GoogleCloudPlatform/gcsfuse/pull/871/files, where Dockerfile will go into master and where there is a bash script(setup_container.sh) to install gcsfuse and run model inside container.

Reply:
Done